### PR TITLE
{bio}[intel/2019a] AUGUSTUS v3.3.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
~#8429~ (SAMtools, HTSlib, BCFtools), ~#8430~ (lpsolve), ~#8431~ (BamTools)

### CODE REVIEW

File: easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.3.2-intel-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+sanity_check_paths = {
+    'files': ['bin/augustus', 'bin/bam2hints', 'bin/espoca', 'bin/etraining',
+              'bin/fastBlockSearch', 'bin/filterBam', 'bin/getSeq', 'bin/homGeneMapping', 'bin/joingenes',
+              'bin/load2sqlitedb', 'bin/prepareAlign'],
+    'dirs': ['config', 'scripts'],
+}
+
+modextrapaths = {'PATH': 'scripts'}
+modextravars = {'AUGUSTUS_CONFIG_PATH': '%(installdir)s/config'}
```
Comment (vanzod) [inline easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.3.2-intel-2019a.eb]: One of the things that we recently discovered is that some of the scripts in `%(installdir)/scripts` require the path in `AUGUSTUS_CONFIG_PATH` to be writable. We are currently working on a way to set this variable to `%(installdir)s/config` only if the user does not have it already defined in his/her own environment.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 6 out of 6 (1 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/5b1e7c9916e99389e102e28ee6cbbda0 for a full test report.

File: easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.3.2-intel-2019a.eb (hunk 1 above)
Comment (vanzod) [reply easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.3.2-intel-2019a.eb]: Let's leave tis improvement for a future PR so that it is no a block for this one.

**Comment by vanzod** [comment]
Re-trigger Travis

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/f06eabfb7276ce9d58a1e99ed4f119bc for a full test report.

**Pull request merged**